webapp/src/pages/createApplicationForm/components/Question.js

```diff
@@ -219,8 +407,27 @@ const Question = forwardRef(({
     }),
   ];
 
+  const validationOptions = [
+    option({
+      value: 'friendly-mode',
+      label: 'Friendly Mode',
```

Comment:
Please label this as "Word Limit" instead of "Friendly mode" so that it's clear that they're setting a max and min word limit.
Also, this text needs to be translated please.